src/Elastic.Markdown/Myst/Renderers/HtmxLinkInlineRenderer.cs

```diff
@@ -74,6 +74,17 @@ protected override void Write(HtmlRenderer renderer, LinkInline link)
 		else
 			base.Write(renderer, link);
 	}
+
+	private static bool HasSameTopLevelSegment(string currentUrl, string targetUrl)
+	{
+		var currentUrlSegments = currentUrl.Split('/', StringSplitOptions.RemoveEmptyEntries);
+		var targetUrlSegments = targetUrl.Split('/', StringSplitOptions.RemoveEmptyEntries);
+
+		if (currentUrlSegments.Length == 0 || targetUrlSegments.Length == 0)
+			return false;
+
+		return currentUrlSegments[0].Equals(targetUrlSegments[0], StringComparison.OrdinalIgnoreCase);
```

Comment:
Isn't `targetUrlSegments[0]` always `docs` ?

Reply:
Ah dang. we should add the `/docs` prefix also for local development.